imgui/src/imgui/imgui_impl_metal.h

```diff
@@ -0,0 +1,28 @@
+// dear imgui: Renderer Backend for Metal
```

Comment:
Should this be included? It is not uses right?

Reply:
Will check. This may have come in from a previous version I was playing with.

Reply:
Removed files.